README.md

```diff
@@ -4,9 +4,10 @@
 
 > Professional Portfolio created using HTML, CSS
 
-## Link to the project 
+## View The Project here
+
+ https://shingiraibhengesa.github.io/My-Portfolio/
```

Comment:
- [ ] [Optional] It would be more professional if you add your deployment link under `Live Demo` title not `View The Project here` like this:

`## Live demo`
   `The live demo [link](https://shingiraibhengesa.github.io/My-Portfolio/).`


and it will look like that

## Live demo
   The live demo [link](https://shingiraibhengesa.github.io/My-Portfolio/).